src/devtoolsPanel.ts

```diff
@@ -179,7 +181,50 @@ export class DevToolsPanel {
             sourceMaps: `${this.config.sourceMaps}`,
         });
 
-        // TODO: Parse message and open the requested file
+        // Parse message and open the requested file
+        const { column, line, url, omitFocus } =
+            JSON.parse(message) as { column: number, line: number, url: string, omitFocus: boolean };

```

Comment:
This feels like something to have a defined interface for.  Casting to an anonymous interface with four members seems excessive.

Reply:
Added a new interface